src/components/ui/form/NumberInput.tsx

```diff
@@ -17,7 +17,7 @@ export interface NumberInputProps
 const NumberInputWrapper = styled.div`
   position: relative;
   width: 100%;
-  min-width: 45px;
+  min-width: 10px;
```

Comment:
I did this to allow for a smaller DMGPalettePicker, and I couldn't figure out a way to override the style. I don't think it causes problems, but I need to do some more digging/testing.